C-Plus-Plus/math/Euler's_Totient_function.cpp

```diff
@@ -0,0 +1,48 @@
+// Euler's Totient Function represented by M(n)
+// For integer n it calculates the no. of coprimes
+// of n from 0 to n. Do refer line 18 for 
```

Comment:
First of all Euler's totient function calculates the number of coprimes of **n** from **1 to n**. Which clearly means that all the numbers between 1 to n (both inclusive) that are relatively prime to **n** are to be only counted.
- The function written by you counts all the coprimes between 0 to n.
Please modify your function to produce required output.

Reply:
@atarax665 Sir, I have removed coprime pairs including 0 and I have updated the definition of Euler's Totient Function definition.

Reply:
> @atarax665 Sir, I have removed coprime pairs including 0 and I have updated the definition of Euler's Totient Function definition. So, I have changed sample output corresponding to it.

@kushagraiitbbs I would like to emphasis on **'relatively prime to n'**, which means only numbers that have common divisor as 1 with n are to be counted.

Example- 
Input = 4
Output = 2
Reason: Only gcd(1, 4) =1 and gcd(3, 4) = 1 will be counted.

Refer to [this](https://www.geeksforgeeks.org/eulers-totient-function/) for better insight.

Reply:
@atarax665 Sir, actually this is the very first code that I have written but you said that answer for 4,6 and 9 are wrong. So, I changed my code to find number of coprime pairs from 1 to n because this is the statement of my issue. Please do check my very first commit.

Reply:
@atarax665 This is the link of my very first commit 
https://github.com/TesseractCoding/NeoAlgo/pull/470/commits/b69525c78d250a2a8707cb20e7ea1c8e565ea8ed

Reply:
> @atarax665 Sir, actually this is the very first code that I have written but you said that answer for 4,6 and 9 are wrong. So, I changed my code to find number of coprime pairs from 1 to n because this is the statement of my issue. Please do check my very first commit.

My bad, actually I was reviewing this PR and #484 together and the function was wrong for #487. I guess I commented the same on your PR too (Though I edited it later). Can you please restore the function ass in your first commit.

Reply:
@atarax665 Sir, I have restored my code with proper indentation and clear definition of Euler's Totient Function at the beginning of code plus sample input and output.

Reply:
Great! Do some minor changes and then it would be good to merge.